### DISCUSSION TITLE
Change for PK profile after adding a new compartment

### DISCUSSION THREAD

**Discussion by zjh112** [Open-Systems-Pharmacology/Forum#1305] [Questions & Problems] [answered]
I was able to run the simulation with added compartment. However the PK profile changed dramatically comparing to before adding the compartment. I attached the model here. Can you please help me to see where I missed？
[adding breast.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9591709/adding.breast.zip)

**Comment by StephanSchaller** [comment]
I can't look into your model at this time, but FYI here is a publication with PK-Sim (Verison 4.2, I believe) on breastfeeding:
https://ascpt.onlinelibrary.wiley.com/doi/10.1038/clpt.2009.151

An here a recent review: https://www.sciencedirect.com/science/article/pii/S0753332220312300

**Comment by wilbertdew** [comment]
I had a short look and I did a few observations:
- the neighborhood from Breast interstitial to VenousBlood Plasma was missing
- the neighborhood from Breast endosome to EndosomalClearance was missing (causing accumulation in the endosome and saturation of FcRn --> more clearance)
- the TwoPoresTransportLink in Passive Transports is not working anymore, I think because a condition was added for the source: "in container: AmnioticFluid". As a consequence, there is no interstitial concentration of Infliximab in any organ.

I could not fix this last part quickly for some reason and I saw there are many more changes between the two simulations so I had to stop here unfortunately. Hopefully this helps a bit.

Best,

Wilbert

**Reply by zjh112** [reply]
Thank you for your reply. I deleted the AmnioticFluid in TwoPoresTransportLink, and it seemed better than before. However, I confronted with a new problem.
![image](https://user-images.githubusercontent.com/112473494/191198924-755746d2-3497-4087-a1ee-49b1d02ee7b6.png)

**Comment by wilbertdew** [comment, marked as answer]
I don't know exactly where this is coming from, but if you replace the formulas in the spatial structure --> neighborhoods --> Pericentral_pls_Pericentral_int for the Peclet numbers for large and small pores, respectively, by:
P_endothelial_l * SA_pls_int > 0 ? (J_iso+alpha_l*Q_lymph)*(1-sigma_l)/(P_endothelial_l*SA_pls_int) : 0
P_endothelial_s * SA_pls_int > 0 ? (-J_iso+(1-alpha_l)*Q_lymph)*(1-sigma_s)/(P_endothelial_s*SA_pls_int) : 0
these formulas are the default formulas, in v11, not sure if it was changed in your model or if the formulas in your model are coming from an older version.
I did this for both Pericentral and Periportal, but probably only Pericentral is enough. This worked for me and finally resulted in a strongly reduced clearance.

**Reply by zjh112** [reply]
Thank you very much. I can run it

**Reply by wilbertdew** [reply]
Great, can you change this topic to "answered" then?
